changes.xml

```diff
@@ -23,6 +23,12 @@
     xsi:schemaLocation="http://maven.apache.org/changes/1.0.0 http://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd">
   <body>
 
+    <release version="2.0.6" date="not released">
+      <action type="update" dev="twolfart">
```

Comment:
please add issue="93" to the action element

Reply:
done